### PR TITLE
fix(dynamodb): pack .mjs files

### PR DESCRIPTION
**Pull request by skyrpex**
Fixes https://github.com/winglang/wing/issues/6005

### CODE REVIEW

**Review by staycoolcall911 — APPROVED** [review]

File: dynamodb/package.json [hunk 1]
```diff
@@ -11,6 +11,10 @@
     "url": "https://github.com/winglang/winglibs.git",
     "directory": "dynamodb"
   },
+  "files": [
+    "*.w",
+    "*.mjs"
+  ],
```
Comment (MarkMcCulloh) [inline dynamodb/package.json:17]: Wing should automatically be adding these as part of the CLI. *.w is already included, and *.mjs should be fixed upstream

Comment (skyrpex) [reply dynamodb/package.json:17]: We need to be careful with the defaults, though. We might end up including config files and other unrelated stuff (ie, eslint config)

Comment (skyrpex) [reply dynamodb/package.json:17]: Maybe we should discuss wing pack altogether? How does it differ from npm pack?

Comment (MarkMcCulloh) [reply dynamodb/package.json:17]: For now it's primarily an opinioned set of defaults/checks. Ideally we don't want to rely on the user knowing all about the various package.json setting they need to handle to publish something.

**Pull request merged**